packaging/microshift/config.yaml

```
@@ -131,6 +131,9 @@ storage:
     # storage driver. Like CSIStorageDriver, uninstallation is not supported as this can lead to orphaned storage
     # objects.
     # Allowed values are: unset, [], or one or more of ["snapshot-controller", "snapshot-webhook"]
+    # example:
+    #   - snapshot-controller
+    # - snapshot-webhook
```

Is this auto-generated?
```suggestion
    # Example:
    # - snapshot-controller
    # - snapshot-webhook
```

---

I know, controller-gen _insists_ on generating it this way. I tried a handful of permutations on the tag's value but it always comes out this way.

---

`ci/prow/verify` failed, I don't think we should include this "suggestions"